src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -291,7 +309,53 @@ public Response listAuthenticatedUsers() {
         return ok(userArray);
     }
 
+    
+    @GET
+    @Path(listUsersPartialAPIPath)
+    @Produces({"application/json"})
+    public Response filterAuthenticatedUsers(@QueryParam("searchTerm") String searchTerm,
+                        @QueryParam("selectedPage") Integer selectedPage,
+                        @QueryParam("numDisplay") Integer itemsPerPage,
+                        @QueryParam("sortKey") String sortKey
+    ) { 
+        System.out.println("_YE_OLDE_QUERY_COUNTER_");
+        
+        boolean DEBUG_MODE = false; // DANGER HERE!  Make sure it's false!
+                
+        if (DEBUG_MODE==true){      // DEBUG: use userIdentifier
+            
+            // DEBUG MODE, pass on by to the actual logic
+           
+        } else if ((session.getUser() != null)&&(session.getUser().isAuthenticated()) && (session.getUser().isSuperuser())){            
```

Comment:
- This should also work with an API token (tests need writing).  
- Extra line gives opportunity to use JSON output in the UI.  This was from initial prototype and worth leaving in as an example if we decided to use the JSON